### ISSUE TITLE
Email case insensitivity on login

### ISSUE THREAD

**Issue by DNGros** [rohitdatta/pepper#12]
The user login email is case sensitive resulting incorrect account-not-found errors or forcing the user to try variations of their email until they find the how they entered it inicially
It could likely be fixed with a change to https://github.com/rohitdatta/pepper/blob/759d7a4051bddc5ec481d4d5bff9d17f355636e2/pepper/users/views.py#L45
to do a case insensitive query.
(this seems fairly easy to do https://stackoverflow.com/questions/16573095/case-insensitive-flask-sqlalchemy-query).

It should also probably be investigated whether the corporate login has this same issue. The password reset link doesn't seem to have this issue, but I am not sure why.

**Comment by rohitdatta** [comment]
Agreed. We're going to change this to instead `.lower()` on account creation, which won't deploy for myHackTX 2017, but will prevent duplicate accounts in the system
